lib/bugsnag/api/client/errors.rb

```diff
@@ -4,72 +4,68 @@ class Client
-        def resolve_error(error, options = {})
-          patch "errors/#{error}", options.merge({:resolved => true})
-        end
-
-        # Re-open an error
+        # @argument ids [(Array<String>/String)] An Id, or array of Ids to update
         #
-        # @param error [String] A Bugsnag error
-        # @return [Sawyer::Resource] The updated error
-        # @see https://bugsnag.com/docs/api/errors#update-an-error-s-status
-        # @example
-        #   Bugsnag::Api.reopen_error("518031bcd775355c48a1cd4e")
-        def reopen_error(error, options = {})
-          patch "errors/#{error}", options.merge({:resolved => false})
+        # @option severity [String] The Error's new severity. One of: info, warning, error
+        # @option assigned_collaborator_id [String] THe collaborator to assign to the Error
+        # @option issue_url [String] Updates to link to an existing 3rd party issue
+        # @option issue_title [String] Updates the issues title
+        # @option reopen_rules [Object] Snooze rules for automatically reopening the Error
+        # @return [Sawyer::Resource] Updated Error
+        # @see http://docs.bugsnagapiv2.apiary.io/#reference/errors/errors/update-an-error
+        def update_errors(project_id, ids, operation, options = {})
+          case ids
+          when String
+            patch "projects/#{project_id}/errors/#{ids}", options.merge({:operation => operation})
+          when Array
+            patch "projects/#{project_id}/errors", options.merge({:operation => operation, :query => {:error_ids => ids.join(' ')}})
+          else
+            raise ArgumentError, "ids must be a String or an Array"
+          end
         end
 
-        # Update an error
+        # Delete an Error
         #
-        # @param error [String] A Bugsnag error
-        # @return [Sawyer::Resource] The updated error
-        # @see https://bugsnag.com/docs/api/errors#update-an-error-s-status
-        # @example
-        #   Bugsnag::Api.update_error("518031bcd775355c48a1cd4e")
-        def update_error(error, options = {})
-          patch "errors/#{error}", options
-        end
-
-        # Delete an error
+        # @argument error_id [String] ID of error to delete (conflicts with project_id)
+        # @argument project_id [String] Id of project to delete all errors from (conflicts with error_id)
         #
-        # @param error [String] A Bugsnag error
-        # @return [Boolean] `true` if error was deleted
-        # @see https://bugsnag.com/docs/api/errors#delete-an-error
-        def delete_error(error, options = {})
-          boolean_from_response :delete, "errors/#{error}", options
+        # @return 
+        # @see http://docs.bugsnagapiv2.apiary.io/#reference/errors/errors/delete-an-error
+        def delete_errors(error_id=nil, project_id=nil, options = {})
+          if !error_id.nil?
+            boolean_from_response :delete, "errors/#{error_id}", options
```

Comment:
Given the previous discussion about the  `error/[id]` endpoint being deprecated, should this variant go away and instead always require the `project_id`?

```diff
- def delete_errors(error_id=nil, project_id=nil, options = {})
+ def delete_errors(project_id, error_id=nil, options ={})
# ...
-   boolean_from_response :delete, "errors/#{error_id}", options
+   boolean_from_response :delete, "projects/#{project_id}/errors/#{error_id}", options
```